index.html

```diff
@@ -36,3 +36,136 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
         <li class="fa fa-facebook fa-2x"></li>
        </ul>
     </section>
+
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(1).png" alt="firstproject">
+        </div>
+        <p class="card-header">Tonic</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(2).png" alt="firsroject">
+        </div>
+        <p class="card-header">Multi-Post<br> Stories</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
```

Comment:
Please use font color and font family for the project title, paragraph, date section, and technologies that are already defined in the design. Kindly use them for all the projects in the work section. Moreover, every project has a border. Please follow the exact design for all the projects.